### PR TITLE
Fix job permissions description(#2611)

### PR DESCRIPTION
**Pull request by SuperYoko**
### What is changed, added or deleted?

Add "Job" permission description,  granted for ROLE: God/Admin/DBA/User

### What is the related PR or file link(s)?

- This PR is translated from: 
- Related issue discussion:[nebula(#2611)](https://github.com/vesoft-inc/nebula/issues/2611)

### CODE REVIEW

**Review by randomJoe211 — DISMISSED** [review]
Great 👍  Please make sure that you have the content tested.

File: docs-2.0/7.data-security/1.authentication/3.role-list.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ The privileges of roles and the nGQL statements that each role can use are liste
   |Read data|Y|Y|Y|Y|Y|`GO`, `SET`, `PIPE`, `MATCH`, `ASSIGNMENT`, `LOOKUP`, `YIELD`, `ORDER BY`, `FETCH VERTICES`, `Find`, `FETCH EDGES`, `FIND PATH`, `LIMIT`, `GROUP BY`, `RETURN`|
   |Write data|Y|Y|Y|Y||`INSERT VERTEX`, `UPDATE VERTEX`, `INSERT EDGE`, `UPDATE EDGE`, `DELETE VERTEX`, `DELETE EDGES`, `DELETE TAG`|
   |Show operations|Y|Y|Y|Y|Y|`SHOW`, `CHANGE PASSWORD`|
-
+  |Job|Y|Y|Y|Y||`SUBMIT JOB COMPACT`、`SUBMIT JOB FLUSH`、`SUBMIT JOB STATS`、`STOP JOB`、`RECOVER JOB`|
```
Comment (foesa-yang) [inline docs-2.0/7.data-security/1.authentication/3.role-list.md:68]: Maybe you need to add a blank line below, there may have some formatting problems.

Comment (SuperYoko) [reply docs-2.0/7.data-security/1.authentication/3.role-list.md:68]: Fixed， thanks

**Review by foesa-yang — APPROVED** [review]
Thank you for the modification.

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**